Dockerfile.amd64

```diff
@@ -30,30 +30,40 @@ RUN apk update && \
         libmount \
         libtbb@testing \
         libtbb-dev@testing \
-        opencv-dev@testing
+        opencv@testing \
+        opencv-dev@testing \
+       	libgomp \
+        libtbb@testing \
+        libtbb-dev@testing \
+        openblas \
+        openblas-dev
 
 WORKDIR /tmp
-ADD ./config.h /opt/sources/config.h
+# ADD ./include/config.h /opt/sources/config.h
 
+# Why do we have this if we have a similar cmake.Modules folder? Should we have a similar approach?
 RUN git clone https://github.com/justusc/FindTBB.git && \
    cp FindTBB/FindTBB.cmake /usr/share/cmake/Modules/
 
-RUN git clone https://github.com/tiny-dnn/tiny-dnn.git && \
-  mkdir -p tiny-dnn/build && \
-  cp /opt/sources/config.h tiny-dnn/tiny_dnn/config.h && \
-  cd tiny-dnn/build && \
-  cmake .. && \
-  make install
-
 RUN git clone https://github.com/eigenteam/eigen-git-mirror.git && \
    mkdir -p eigen-git-mirror/build && \
    cd eigen-git-mirror/build && \
    cmake .. && \
    make install
-RUN echo "@testing http://nl.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
-RUN apk update && \
-    apk --no-cache add \
-opencv@testing
+
+RUN git clone https://github.com/tiny-dnn/tiny-dnn.git && \
```

Comment:
Here, you should only clone the latest master instead of the entire project history (--depth 1)